src/components/SolanaAlert/SolanaAlert.tsx

```diff
@@ -0,0 +1,89 @@
+import { ChainId } from '@lifi/types';
```

Comment:
@dennyscode can you rename differently than "SolanaAlert" as it is not a quickfix anymore, please call it "SystemAlert" or "InformationAlert"